pkg/api/globalquery/client/client.go

```diff
@@ -1,20 +1,36 @@
 package client
 
 import (
+	"bytes"
 	"context"
+	"errors"
+	"net/http"
 
 	"github.com/els0r/goProbe/pkg/api"
 	"github.com/els0r/goProbe/pkg/api/client"
 	"github.com/els0r/goProbe/pkg/query"
 	"github.com/els0r/goProbe/pkg/results"
+	"github.com/els0r/telemetry/logging"
 	"github.com/fako1024/httpc"
+	jsoniter "github.com/json-iterator/go"
 )
 
 // Client denotes a global query client
 type Client struct {
 	*client.DefaultClient
 }
 
+// SSEClient is a global query client capable of streaming updates
+type SSEClient struct {
```

Comment:
Nitpick: Maybe move the SSE stuff to `streaming.go` as it better clusters the methods...